specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -94,13 +94,7 @@
         ],
         "responses": {
           "200": {
-            "description": "Resource Updated. Returns the network security perimeter.",
-            "schema": {
-              "$ref": "#/definitions/NetworkSecurityPerimeter"
-            }
-          },
-          "201": {
-            "description": "Resource Created. Returns the network security perimeter.",
+            "description": "Resource created. Returns the network security perimeter.",
             "schema": {
               "$ref": "#/definitions/NetworkSecurityPerimeter"
```

Comment:
200 should be for new resource and 201 for re-put on existing resource. Why are you removing the 201 status code ?

Reply:
Reverted the change for PUT call.